yarn.lock

```diff
@@ -5178,9 +5178,9 @@ camelize@^1.0.0:
   integrity sha1-FkpUg+Yw+kMh5a8HAg5TGDGyYJs=
```

Comment:
This should be updated in a separate PR.

Reply:
I don't think caniuse is ever really important enough to merit its own pr -- too much process for its value

Reply:
prefer people bump it whenever they notice the message